docs/guides/extensions/hooks-validate-number-twilio.md

```diff
@@ -29,10 +29,15 @@ npm install twilio
 
 ## Build the Hook
 
-Create a collection called Customers with a text field called `phone_number`. This hook will be used to validate the
-item when a record is saved.
+Create a collection called Customers with a text field called `phoneNumber`. This hook will be used to validate the item
```

Comment:
```suggestion
Create a collection called Customers with a text field called `phone_number`. This hook will be used to validate the item
```

I dont think we should start using pascalCase for database fields in the docs.

Reply:
Will revert 👍